cmd/process.go

```diff
@@ -85,6 +107,12 @@ func Start(a *AppState) *cobra.Command {
 			}
 
 			<-cmd.Context().Done()
+			// clean up
+			time.Sleep(20 * time.Millisecond)
+			for _, c := range registeredDomains {
+				fmt.Printf("\n%s: latest-block: %d last-flushed-block: %d", c.Name(), c.LatestBlock(), c.LastFlushedBlock())
+				c.CloseClients()
+			}
```

Comment:
If the 20ms sleep is required, I wonder if a defer func() for this is better

Reply:
The sleep was originally to allow other go routines finish as I was seeing errors when terminating. However that seems to be fixed now. 
Removed the sleep, also moved to defer 👍